recipes/perl-file-spec/3.48_01/meta.yaml

```diff
@@ -22,7 +22,10 @@ requirements:
   run:
     - perl
 
-# test:
+  test:
+    commands:
+      - perl -v
+
```

Comment:
Don't modify random recipes as part of your PR. If linting fails, just:

    git checkout master
    git pull upstream master
    git checkout pgcgap
    git merge master

Reply:
Ok, sorry about that, thanks!